reconcile/utils/oc.py

```diff
@@ -514,6 +522,32 @@ def get_owned_pods(self, namespace, resource):
 
         return owned_pods
 
+    @staticmethod
+    def get_pod_owned_pvc_names(pods):
```

Comment:
Type hints please? It'll make things easier for me to understand. In this case I believe it's:
```suggestion
    def get_pod_owned_pvc_names(pods: Iterable[dict[str, dict]]) -> set[str]:
```
(but double-check with `tox -e type`)

Reply:
thanks!